deps/chakrashim/src/v8booleanobject.cc

```diff
@@ -23,10 +23,12 @@
 namespace v8 {
 
 using jsrt::ContextShim;
+using jsrt::IsolateShim;
 
-Local<Value> BooleanObject::New(bool value) {
```

Comment:
Maybe using deprecated APIs only result in a build warning by default, not failure?